entity-service-change-event-api/build.gradle.kts

```diff
@@ -0,0 +1,68 @@
+import com.google.protobuf.gradle.generateProtoTasks
+import com.google.protobuf.gradle.id
+import com.google.protobuf.gradle.ofSourceSet
+import com.google.protobuf.gradle.plugins
+import com.google.protobuf.gradle.protobuf
+import com.google.protobuf.gradle.protoc
+
+plugins {
+  `java-library`
+  id("com.google.protobuf") version "0.8.15"
+  id("org.hypertrace.publish-plugin")
+}
+
+val generateLocalGoGrpcFiles = false
+
+protobuf {
+  protoc {
+    artifact = "com.google.protobuf:protoc:3.15.6"
+  }
+  plugins {
+    // Optional: an artifact spec for a protoc plugin, with "grpc" as
+    // the identifier, which can be referred to in the "plugins"
+    // container of the "generateProtoTasks" closure.
+    id("grpc_java") {
+      artifact = "io.grpc:protoc-gen-grpc-java:1.40.1"
+    }
+
+    if (generateLocalGoGrpcFiles) {
+      id("grpc_go") {
+        path = "<go-path>/bin/protoc-gen-go"
+      }
+    }
+  }
+  generateProtoTasks {
+    ofSourceSet("main").forEach {
+      it.plugins {
+        // Apply the "grpc" plugin whose spec is defined above, without options.
+        id("grpc_java")
+
+        if (generateLocalGoGrpcFiles) {
+          id("grpc_go")
+        }
+      }
+      it.builtins {
+        java
+
+        if (generateLocalGoGrpcFiles) {
+          id("go")
+        }
+      }
+    }
+  }
+}
+
+sourceSets {
+  main {
+    java {
+      srcDirs("src/main/java", "build/generated/source/proto/main/java", "build/generated/source/proto/main/grpc_java")
```

Comment:
"src/main/java" is default, it's being duplicated here (that will lead to dupe protos which I think you mentioned seeing)

Reply:
This is same as entity service protos. Doesn't fix the issue.

Reply:
entity api file is wrong, at a minimum this is a redundant declaration, but this has been the issue in other places (due to that redundancy).

Reply:
Removed